npc/re/custom/lasagna/lasagna_npcs_2.txt

```diff
@@ -0,0 +1,2212 @@
+			next;
+			mes "[Sleepy meow]";
+			mes "What is that look in your eyes? Do you think I'm pathetic meow?";
+			next;
+			mes "[Sleepy meow]";
+			mes "That's rude but It's okay. You're not the only one meow.";
+			next;
+			mes "[Sleepy meow]";
+			mes "It's okay if you decide not to help meow.";
+			next;
+			mes "[Sleepy meow]";
+			mes "Even if I don't die here fighting against sleep, my dad, I mean, Chef, will kill me. Merely over mushroom spores meow.";
+			next;
+			mes "[Sleepy meow]";
+			mes "That's okay, too. I came back to life many times already. Long and deep sleep is all I need to revive meow.";
+			next;
+			mes "[Sleepy meow]";
+			mes "So, please think of it as... saving someone's life...and help meeee...";
+			setquest 14591;// Secret Ingredient
+			close;
+		case 0:
+		case 1:
+			break;
+		case 2:
+			if (countitem(921) < 5)
+				break;
+			mes "[Sleepy meow]";
+			mes "Adventurer...? Are you waking me up cuz you're done? Let me see...";
+			next;
+			mes "[Sleepy meow]";
+			mes "Wow, you're really done! That was so quick of you that I am wide awake. You really are a wonder!";
+			next;
+			mes "[Sleepy meow]";
+			mes "Why don't you work at Pub-Oven instead of me? Then again, I'm sure you'd hate that, too.";
+			next;
+			mes "[Sleepy meow]";
+			mes "I might need to get some mushroom spores or something tomorrow as well, so drop by again. I'm going to bring these quickly and relax. Thanks! I owe you.";
+			erasequest 14591;// Secret Ingredient
+			delitem 921,5;// Mushroom Spore
+			setquest 14592;// Resting Time
+			getexp 500,1000;
+			close;
+		}
+		mes "[Sleepy meow]";
+		mes "Have you finished it already meow?";
+		next;
+		mes "[Sleepy meow]";
+		mes "Of course not. This is not it. Did you forget already meow?";
+		next;
+		mes "[Sleepy meow]";
+		mes "Catch ten Trans Spores and bring five mushroom spores meow.";
+		next;
+		mes "[Sleepy meow]";
+		mes "Call me when you're done. Don't wake me up for no reason meow.";
+		close;
+	case 0:
+	case 1:
+		mes "[Sleepy meow]";
+		mes "there's no more for today, so can go...";
+		next;
+		mes "[Sleepy meow]";
+		mes "don't disturb my... sleep...";
+		close;
+	case 2:
+		mes "[Sleepy meow]";
+		mes "An adventurer. The adventurer from yesterday. Great, please save me today as well meow.";
+		next;
+		mes "[Sleepy meow]";
+		mes "There seems to be another bothersome fish cake order that came in. So please get the ingredients for me. I'm busy today as well, desperately fighting against the sunlight meow.";
+		next;
+		mes "[Sleepy meow]";
+		mes "Catch ten Trans Spores and bring them five mushroom spores meow.";
+		next;
+		mes "[Sleepy meow]";
+		mes "Please...help me...";
+		erasequest 14592;// Resting Time
+		setquest 14591;// Secret Ingredient
+		close;
+	}
+
+OnInit:
+	questinfo 14591, QTYPE_QUEST, 1;
+	setquestinfo_req 14591, 14592,0;
+	setquestinfo_level 14591,20,20;
+	end;
+}
+
+// todo : quest depends on player level
+lasagna,222,187,3	script	Chef#dorcon	4_CAT_CHEF,{
```

Comment:
This NPC.
Problem:
https://i.imgur.com/VLwaKbX.png

tl;tr
i did finish the same quest many many times.
After reading code and description i expect if i finish one time the quest, i can't take new one and finish it in next 4 hours. But for some reason i already did it three times. That means the quest has disappear after some time or after some actions (i don't know which one, because i have test all doram-story quests). But i notice some bug inside, not sure where and how reproduce, but the thing that i have finished the quest 3x times in less then 1-2 hours = strange.